### ISSUE TITLE
Adding more detailled descriptions to fields?

### ISSUE THREAD

**Issue by Stefan-Heimersheim** [EGaraldi/corecon#11]
Hello,

the descriptions of many fields are very short such as

- `ionized_fraction` :`$x_\mathrm{HII}$`
- `Lya_flux_ps` :`$P\, [\mathrm{km/s}]$`
- `mfp` :`$\lambda_\mathrm{mfp} \, [h^{-1} \, \mathrm{Mpc}]$`
- `tau_eff_HI` :`$\tau_\mathrm{eff, HI}$`
- `tau_eff_HeII` :`$\tau_\mathrm{eff, HeII}$`
- `eta` :`$\eta$`
- `qlf` :`$\log_{10} (phi_\mathrm{QSO}) \, [\mathrm{Mpc}^{-3} \, \mathrm{mag}^{-1}]$`
- `glf` :`$\log_{10} (phi_\mathrm{GAL}) \, [\mathrm{Mpc}^{-3} \, \mathrm{mag}^{-1}]$`
- `T0` :`T0 [K]`
- `tau_cmb` :`$\tau_\mathrm{CMB}$`
- `sfrd` :`$\log \Psi \, [\mathrm{M}_\odot \mathrm{yr}^{-1} \mathrm{Mpc}^{-3}]$`
- `Lya_spike_galaxy_correlation` :`$\langle f \rangle / \bar{f} - 1$`

is this intended (i.e. are they just supposed to be TeX axis labels) or should they be more descriptive?

In the latter case, the list [in the docs](https://corecon.readthedocs.io/en/latest/#available-constraints) contains more detailed descriptions such as `IGM temperature at mean density` or `CMB optical depth`.
One could add even more information such as `CMB optical depth due to free electrons, integrated from redshift 0 to 50` unless that is already too much.

Edit: The individual fields usually don't give any information on the variable either, such as this one for the optical depth: `Cosmological parameter result from the Planck temperature and lensing-potential power spectra measurements of the CMB anisotropies.` Of course it's not their purpose to explain the variable, so that's fine. But we should maybe explain somewhere what the variable is.

Best,
Stefan

**Comment by EGaraldi** [comment]
Hi, 
yes these descriptors are essentially the field symbol in latex. The idea is to use them e.g. to make labels for plots. To mae things clearer I've renamed them as "symbol". Concerning the description for individual entries: while I think the target audience of this package is researcher in the field, that therefore should know well what we are talking about, I am all for being extra clear. For now I have created for each field a variable field_description which contains a short description of what is stored in the field (used to be the internal variable "header")
